src/components/atoms/LogoText.jsx

```diff
@@ -46,6 +46,7 @@ const LogoTextContainer = styled.div`
     color: ${({ color }) => color};
     display: inline-block;
     font-size: 1em;
+    font-family: ${({ fontFamily }) => `${fontFamily}`};
```

Comment:
I don't think template literal is needed here, can be just ${({ fontFamily }) => fontFamily}

But all good! 👍